sdk/servicebus/Azure.Messaging.ServiceBus/src/Transports/Amqp/AmqpTransportBody.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Azure.Messaging.ServiceBus.Transports.Amqp
```

Comment:
I think this can probably go in Azure.Messaging.ServiceBus.Amqp, and in the existing Amqp directory. If we get to a point where we have multiple protocols we can introduce a Transports top level directory.

Reply:
+1

Reply:
Sounds good. Will change.

Reply:
Done.